packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -21,6 +22,8 @@ const MenuLink = ({
     text,
     onClickLink,
 }) => {
+    const derivStaticUrl = /(deriv.com)/.test(link_to);
```

Comment:
```suggestion
    const deriv_static_url = /(deriv.com)/.test(link_to);
```
Naming convention. This will match anything with `deriv.com` in it. The following non static platforms will match this regex:

```
app.deriv.com
bot.deriv.com
deriv.com
```

Best to use a more specific regex if you want to only match our static sites. Something like:
```
^(https:\/\/deriv.(com|be))
```
Check if it matches your requirements though

Reply:
For now, the current links that we redirect is to deriv.com. So i think this is fine.